src/main/java/com/robertx22/mine_and_slash/event_hooks/ontick/OnServerTick.java

```diff
@@ -128,6 +128,37 @@ public static void onEndTick(ServerPlayer player) {
                     playerData.spellCastingData.charges.onTicks(player, 5);
                 }
 
+                // Every second, apply passive decay to inactive threshold progress
+                if (age % 20 == 0) {
+                    long now = player.level().getGameTime();
+                    var unit = Load.Unit(player);
+                    if (unit != null) {
+                        // Iterate only active keys per resource
+                        for (var rt : com.robertx22.mine_and_slash.saveclasses.unit.ResourceType.values()) {
```

Comment:
Use an import instead
```suggestion
                        for (var rt : ResourceType.values()) {
```